docs/blog/_posts/2019-03-04-13th-dotty-milestone-release.md

```diff
@@ -0,0 +1,439 @@
+---
+layout: blog-page
+title: Announcing Dotty 0.13.0-RC1 with Spark support, top level definitions and redesigned implicits
+author: Aggelos Biboudis
+authorImg: /images/aggelos.jpg
+date: 2019-03-04
+---
+
+Hello hello! This is the second release for 2019. Spark, top level definitions
+and redesigned implicits ✨🎊🎉 are the most important inclusions in this release
+and you will understand why we are super excited, in a bit!
+
+Without further ado, today we release the version 0.13.0-RC1 of the Dotty
+compiler. This release serves as a technology preview that demonstrates new
+language features and the compiler supporting them.
+
+Dotty is the project name for technologies that are being considered for
+inclusion in Scala 3. Scala has pioneered the fusion of object-oriented and
+functional programming in a typed setting. Scala 3 will be a big step towards
+realising the full potential of these ideas. Its main objectives are to
+
+- become more opinionated by promoting programming idioms we found to work well,
+- simplify where possible,
+- eliminate inconsistencies and surprising behaviours,
+- build on strong foundations to ensure the design hangs together well,
+- consolidate language constructs to improve the language’s consistency, safety, ergonomics, and
+  performance.
+
+You can learn more about Dotty on our [website](https://dotty.epfl.ch).
+
+<!--more-->
+
+This is our 13th scheduled release according to our
+[6-week release schedule](https://dotty.epfl.ch/docs/usage/version-numbers.html).
+
+# What’s new in the 0.13.0-RC1 technology preview?
+
+## Experimental support for Spark
+
+Dotty projects have always been able to [depend on Scala 2
+libraries](https://github.com/lampepfl/dotty-example-project#getting-your-project-to-compile-with-dotty),
+and this usually works fine as long as the Dotty code does not call a Scala 2
+macro directly. However, [Spark](http://spark.apache.org/) was known to not work

```

Comment:
Parenthesize "`as long as the Dotty code does not call a Scala 2 macro directly`", as it is not an essential part of that sentence, and it currently confuses the point you're trying to get across.